x-pack/test/functional/apps/machine_learning/index.ts

```diff
@@ -19,6 +20,26 @@ export default function({ getService, loadTestFile }: FtrProviderContext) {
     after(async () => {
       await ml.securityCommon.cleanMlUsers();
       await ml.securityCommon.cleanMlRoles();
+
+      await ml.testResources.deleteSavedSearches();
+
+      await ml.testResources.deleteIndexPattern('ft_farequote');
+      await ml.testResources.deleteIndexPattern('ft_ecommerce');
+      await ml.testResources.deleteIndexPattern('ft_categorization');
+      await ml.testResources.deleteIndexPattern('ft_event_rate_gen_trend_nanos');
+      await ml.testResources.deleteIndexPattern('ft_bank_marketing');
+      await ml.testResources.deleteIndexPattern('ft_ihp_outlier');
+      await ml.testResources.deleteIndexPattern('ft_egs_regression');
+
+      await esArchiver.unload('ml/farequote');
+      await esArchiver.unload('ml/ecommerce');
+      await esArchiver.unload('ml/categorization');
+      await esArchiver.unload('ml/event_rate_nanos');
+      await esArchiver.unload('ml/bm_classification');
+      await esArchiver.unload('ml/ihp_outlier');
+      await esArchiver.unload('ml/egs_regression');
```

Comment:
Not sure `ciGroup3 ` should be responsible for this clean-up 🤔

Reply:
As discussed on slack: ciGroup3 holds a couple test suites and each of them is repsonsible for the own cleanup. That's why we make sure to clean everything up, before we leave the ML UI / ML API / Transform UI test suite.